<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MvlcConnectWidget</class>
 <widget class="QWidget" name="MvlcConnectWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>374</width>
    <height>91</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MvlcConnectWidget</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QTabWidget" name="tabs_connectMethod">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab_usb">
      <attribute name="title">
       <string>USB</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="0,1,0,0">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>USB Device</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="combo_usb">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pb_refresh">
         <property name="toolTip">
          <string>Refresh serial ports</string>
         </property>
         <property name="statusTip">
          <string>Refresh serial ports</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="resources.qrc">
           <normaloff>:/connecting.png</normaloff>:/connecting.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>16</width>
           <height>16</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pb_connect_usb">
         <property name="text">
          <string>Connect</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_eth">
      <attribute name="title">
       <string>Ethernet</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="0,1,0">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Hostname / Ip</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="combo_eth">
         <property name="minimumSize">
          <size>
           <width>150</width>
           <height>0</height>
          </size>
         </property>
         <property name="editable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pb_connect_eth">
         <property name="text">
          <string>Connect</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
